### PR TITLE
Adding the anydb hosts to output.json and to the hosts files

### PR DESCRIPTION
**Pull request by KimForss**
### Problem

In order to be able to run the correct ansible workbooks against the anydb servers we need to add them to the output files

### Solution

Include the anydb nodes in the output_files module to generate:

[ase]

[oracle]
10.7.2.10  ansible_connection=ssh  ansible_user=azureadm
10.7.2.11  ansible_connection=ssh  ansible_user=azureadm

[db2]

[sqlserver]

and

asenodes:
      hosts:

    oraclenodes:
      hosts:
        10.7.2.10:
          ansible_connection:  "ssh"
          ansible_user:        "azureadm"
        10.7.2.11:
          ansible_connection:  "ssh"
          ansible_user:        "azureadm"

    db2nodes:
      hosts:

    sqlservernodes:
      hosts:

### CODE REVIEW

File: deploy/terraform/modules/output_files/inventory.tf [hunk 1]
```diff
@@ -58,6 +58,27 @@ resource "local_file" "output-json" {
       }
       }
       if database != {}
+    ],
+        "anydatabases" = [for database in local.anydatabases : {
```
Comment (rukawata) [inline deploy/terraform/modules/output_files/inventory.tf:62]: nit: indention is off.

File: deploy/terraform/modules/output_files/inventory.tf [hunk 2]
```diff
@@ -58,6 +58,27 @@ resource "local_file" "output-json" {
...
+      platform          = database.platform,
+      db_version        = database.db_version,
+      os                = database.os,
+      size              = database.size,
+      filesystem        = database.filesystem,
+      high_availability = database.high_availability,
+      authentication    = database.authentication,
+      credentials       = database.credentials,
+      nodes = [for ip-anydbnode in local.ips-anydbnodes : {
+        dbname       = local.anydb_vms[index(local.ips-anydbnodes, ip-anydbnode)].name
```
Comment (rukawata) [inline deploy/terraform/modules/output_files/inventory.tf:72]: This should be the `computer_name`/`hostname` of the VM not the VM resource name.

File: deploy/terraform/modules/output_files/inventory.tf (hunk 1 above)
Comment (KimForss) [reply deploy/terraform/modules/output_files/inventory.tf:62]: fixed

File: deploy/terraform/modules/output_files/inventory.tf (hunk 2 above)
Comment (KimForss) [reply deploy/terraform/modules/output_files/inventory.tf]: It is now - it is either based on what is passed in or computed.

File: deploy/terraform/modules/output_files/inventory.tf [hunk 3]
```diff
@@ -59,6 +59,28 @@ resource "local_file" "output-json" {
...
     ],
+    "anydatabases" = [for database in local.anydatabases : {
+      platform          = database.platform,
+      db_version        = database.db_version,
+      os                = database.os,
+      size              = database.size,
+      filesystem        = database.filesystem,
+      high_availability = database.high_availability,
+      authentication    = database.authentication,
+      credentials       = database.credentials,
```
Comment (rukawata) [inline deploy/terraform/modules/output_files/inventory.tf:70]: Default credentials section is missing in local_variable.tf.
Therefore when deploying without crendentials defined, see below error.
```
Error: Unsupported attribute

  on deploy/terraform/modules/output_files/inventory.tf line 70, in resource "local_file" "output-json":
  70:       credentials       = database.credentials,

This object does not have an attribute named "credentials".

```

File: deploy/terraform/modules/output_files/inventory.tf [hunk 4]
```diff
@@ -59,6 +59,28 @@ resource "local_file" "output-json" {
       }
       if database != {}
     ],
+    "anydatabases" = [for database in local.anydatabases : {
```
Comment (rukawata) [inline deploy/terraform/modules/output_files/inventory.tf:62]: databases section above should be resued for both HDB and anyDB.
Can we somehow not to have a seperate anydatabases section in the output.json?

File: deploy/terraform/modules/output_files/inventory.tf (hunk 3 above)
Comment (KimForss) [reply deploy/terraform/modules/output_files/inventory.tf:70]: Fixed

File: deploy/terraform/modules/output_files/inventory.tf (hunk 4 above)
Comment (KimForss) [reply deploy/terraform/modules/output_files/inventory.tf]: Merged - let's see how it works

**Pull request merged**